earthly/rust/scripts/std_checks.py

```diff
@@ -90,7 +90,7 @@ def main():
     results.add(
         vendor_files_check.toml_diff_check("/stdcfgs/clippy.toml", "clippy.toml")
     )
-    results.add(vendor_files_check.toml_diff_check("/stdcfgs/deny.toml", "deny.toml"))
+    results.add(vendor_files_check.toml_diff_check("/stdcfgs/deny.toml", "deny.toml", strict=False))
```

Comment:
No, i want strict checking of cargo-deny config.  Projects should not be able to modify it to suit themselves.